stgit/commands/common.py

```diff
@@ -483,6 +483,14 @@ def __decode_header(header):
             payload = part.get_payload(decode=True)
             msg_data += payload
 
+    # RFC-compliant mail uses CRLF.  Replace this with LF unless
+    # otherwise instructed.
+    if not getattr(options, 'keepcr', False):
+        msg_data_lf = b''
+        for line in msg_data.splitlines():
+            msg_data_lf += line + b'\n'
+        msg_data = msg_data_lf
```

Comment:
Why not `msg_data = msg_data.replace(b'\r\n', b'\n')`?

Reply:
As you can probably tell, python isn't my main language. :smile: This is adapted from a code snipped on Stackoverflow.

I had pondered why the commenter had used `splitlines` rather than `replace`.  One potential advantage is that the built-in `splitlines` should automatically handle *any* alternate line endings, not just CRLF.  It could also potentially be optimized due to the fact that it's not a general-purpose string replacement.

I don't know if either of those will turn out to be real advantages in practice, but in general, using a core library to do the thing you want rather than implementing it yourself seems a good idea, so I kept it that way.